proxies/live/apiproxy/targets/apim.xml

```diff
@@ -1,6 +1,9 @@
 <TargetEndpoint name="apim">
   <PreFlow/>
   <Flows>
+    <Flow name="No-Auth">
```

Comment:
I think your naming pattern is a nicer way to do this, so do you want to update the status flow underneath to something like `APIKeyAuthenticated`?

Reply:
i think the names below were copied from pds, happy to change